apps/browser/qml/pages/components/PrivacySettingsConfirmDialog.qml

```diff
@@ -0,0 +1,147 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import Sailfish.Browser 1.0
+
+Dialog {
+
+    property alias historyEnabled: historyItem.visible
+    property alias cookieAndSiteDataEnabled: cookieAndSiteDataItem.visible
+    property alias passwordsEnabled: passwordsItem.visible
+    property alias cacheEnabled: cacheItem.visible
+    property alias bookmarksEnabled: bookmarksItem.visible
+    property alias sitePermissionsEnabled: sitePermissionsItem.visible
+
+    property int historyPeriod
+
+    property int _siteDataUsage
+    property int _cacheUsage
+
+    SilicaFlickable {
+        anchors.fill: parent
+        contentHeight: column.height
+
+        Column {
+            id: column
+
+            width: parent.width
+
+            spacing: Theme.paddingMedium
+
+            DialogHeader {
+                id: header
+
+                //% "Are you sure you want to clear the browser data?"
+                title: qsTrId("sailfish_browser-ti-confirm_clearing_browser_data");
+                //% "Confirm"
+                acceptText: qsTrId("sailfish_browser-he-confirm");
```

Comment:
Just one small change request from Martin. Could "Confirm" please be changed to "Clear"?
```suggestion
                //% "Clear"
                acceptText: qsTrId("sailfish_browser-he-clear");
```